src/odemis/acq/stream/_sync.py

```diff
@@ -454,13 +470,41 @@ def _getSpotPositions(self):
         logging.debug("Generating points in the SEM area %s, from rep %s and roi %s",
                       lim_main, rep, roi)
 
-        pos = numpy.empty((rep[1], rep[0], 2), dtype=float)
+        pos = numpy.empty((rep[1], rep[0], 2), dtype=float) # in SEM px (at scale = 1)
         posy = pos[:, :, 1].swapaxes(0, 1)  # just a view to have Y as last dim
         posy[:, :] = numpy.linspace(lim_main[1], lim_main[3], rep[1])
         # fill the X dimension
         pos[:, :, 0] = numpy.linspace(lim_main[0], lim_main[2], rep[0])
+
         return pos
 
+    def _getLeftTopPositionPhys(self) -> Tuple[float, float]:
+        """
+        Compute the position of the top-left pixel of the RoA in physical coordinates (m).
+        Note that this is *not* the top-left corner of the image. It is shifted by half a pixel.
+        :return: x, y position of the center of pixel (m,m) in absolute coordinates
+        """
+        rep = tuple(self.repetition.value)
+        roi = self.roi.value
+        width = (roi[2] - roi[0], roi[3] - roi[1])
+
+        # Take into account the "border" around each pixel
+        pxs_rel = (width[0] / rep[0], width[1] / rep[1])
+        lim = (roi[0] + pxs_rel[0] / 2, roi[1] + pxs_rel[1] / 2)
+
+        shape = self._emitter.shape
+        # convert into SEM translation coordinates: distance in px from center
+        # (situated at 0.5, 0.5), can be floats
+        pos_lt = (shape[0] * (lim[0] - 0.5), shape[1] * (lim[1] - 0.5))
+
+        # Convert to physical coordinates
+        epxs = self._emitter.pixelSize.value  # scanner pxs at scale = 1
+        shift_tl_phys = (pos_lt[0] * epxs[0], -pos_lt[1] * epxs[1])  # in m
```

Comment:
No. "tl" stands for "top-left" which is more English."lt" stands for "left-top" which is more logical because the first value is in X and second value in Y. In general, nowadays I use "lt". I'll change the name.